src/config.js

```diff
@@ -256,7 +257,7 @@ export default class Config {
     this.modulesFolder = opts.modulesFolder;
     this.globalFolder = opts.globalFolder || constants.GLOBAL_MODULE_DIRECTORY;
     this.linkFolder = opts.linkFolder || constants.LINK_REGISTRY_DIRECTORY;
-    this.production = !!opts.production;
+    this.production = !!opts.production && opts.production !== 'false';
```

Comment:
It is just weird that we assign same property twice in different ways.
Better to have all the production override logic in one place.